api/api_public_blockchain.go

```diff
@@ -433,6 +453,26 @@ func FormatLogs(logs []vm.StructLog) []StructLogRes {
 	return formatted
 }
 
+// RPCOutputHeader converts the given header to the RPC output.
+func RPCOutputHeader(head *types.Header) (map[string]interface{}, error) {
```

Comment:
The output format is the same with json tags of `Header` struct. Why did you adds this converting function?

Reply:
In some cases, there may be additional information that I want to include or hide, so I usually define the output model like rpcOutputBlock. But this case, it is unnecessary. I'll remove this code and use json tag of Header! 
Thank you aidan!